internal/cli/down.go

```diff
@@ -21,7 +20,8 @@ type downCmd struct {
 func (c *downCmd) register() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "down",
-		Short: "delete kubernetes resources",
+		Short: "delete resources created by 'tilt up'",
+		Args:  cobra.NoArgs,
```

Comment:
hang on, this should be able to take args -- e.g. if someone is using the `--file` arg to specify a non-standard Tiltfile path, or if you only `tilt up`'d services A and B, and so only want to `tilt down` those specific ones, lest you get an error.

Reply:
re: the --file arg

this still works fine. cobra.NoArgs only applies to positional args

https://godoc.org/github.com/spf13/cobra#PositionalArgs

re: if you only tilt up'd services A and B, and so only want to tilt down those specific ones

that seems reasonable, but that doesn't work in the current code. do you think we should file a feature request for it now, or when someone requests it?

Reply:
ah, gotcha. yeah let's wait until someone complains!